environmentManager/index.js

```diff
@@ -0,0 +1,73 @@
+const settings = require('../utils/parserConfigs')();
+const { validEnvironment } = require('../utils/parserConfigs/validators');
+const { getSplitFactory } = require('../sdk');
+
+class EnvironmentManager {
+  
+  constructor() {
+    
+    // In environments are the apiKey, factory an clientReadiness status related to an authToken
+    this._environments = {};
+    
+    // Ready promises for each client in environment manager
+    this._readyPromises = [];
+    
+    this._initializeEnvironments(JSON.parse(process.env['SPLIT_EVALUATOR_ENVIRONMENTS']));
+    
+  }
+  
+  _initializeEnvironments(environmentConfigs){
+    
+    environmentConfigs.forEach(environment => {
```

Comment:
Idem. What if `JSON.parse(process.env['SPLIT_EVALUATOR_ENVIRONMENTS']` is not an array?

Reply:
I created a environmentConfig validator to valid that it is an array and added the validation isnullOrEmpty